run.go

```diff
@@ -150,6 +156,17 @@ var commandRun = cli.Command{
 			Usage:  "output metrics to an external data store (format: type=uri)",
 			EnvVar: "K6_OUT",
 		},
+		cli.StringFlag{
+			Name: "summary-file",
```

Comment:
I like the name `summary` or `report` better here; also give it a shorthand flag!